include/api/CDataFrameAnalyzer.h

```diff
@@ -112,7 +111,6 @@ class API_EXPORT CDataFrameAnalyzer {
     TStrVec m_FieldNames;
     TTemporaryDirectoryPtr m_DataFrameDirectory;
     TJsonOutputStreamWrapperUPtrSupplier m_ResultsStreamSupplier;
-    TDataSearcherUPtrSupplier m_DataSearcher;
```

Comment:
Can we also remove the `TDataSearcherUPtrSupplier` and `#include <core/CDataSearcher.h>`?

Reply:
Done